pages/linux/netcat.md

```diff
@@ -0,0 +1,32 @@
+# nc
+
+> The `nc` (or `netcat`) utility is used for just about anything under the sun involving TCP, UDP, or UNIX-domain sockets. It can open TCP connections, send UDP packets, listen on arbitrary TCP and UDP ports, do port scanning.
```

Comment:
What about a simpler style like this?
```suggestion
> The `nc` (or `netcat`) utility is used for TCP, UDP and UNIX-domain sockets: it can open TCP connections, send UDP packets, listen on arbitrary TCP and UDP ports, do port scanning.
```